apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/ir/IrBuilder.kt

```diff
@@ -584,10 +588,12 @@ internal fun GQLValue.toIrValue(): IrValue {
  * - False
  * - (!)Variable
  * - (!)Variable & (!)Variable
+ *
+ * TODO: rename to toIncludeBooleanExpression
```

Comment:
Same here, naming is hard :) But need to distinguish this one that is about include/skip and the other one which is this one + defer.